velox/functions/prestosql/tests/InPredicateTest.cpp

```diff
@@ -692,3 +692,15 @@ TEST_F(InPredicateTest, floatDuplicateWithFloatingPointRange) {
   auto result = evaluate<SimpleVector<bool>>(predicate, input);
   assertEqualVectors(expected, result);
 }
+
+TEST_F(InPredicateTest, floatDuplicateWithBigintValuesUsingHashTable) {
```

Comment:
Can we add a test for the case the filter is actually a range filter (i.e. one value)?